[improve][doc] add notes to wireshark scripts README (#18563)
apache/pulsar: wireshark/README.md

-->

> > **Note**: This Lua script may cause a crash in the newest version of Wireshark, see [#18439](https://github.com/apache/pulsar/issues/18439). Please make sure the version of Wireshark is lower than v4.0.0.

Reviewer: ```suggestion
> **Note**: This Lua script may cause a crash in the newest version of Wireshark, see [#18439](https://github.com/apache/pulsar/issues/18439). Make sure the version of Wireshark is earlier than v4.0.0.
```

1. Do not use `please` and `thank you` in technical information. Technical information requires an authoritative tone. Terms of politeness are superfluous, convey the wrong tone for technical material, and are not regarded the same way in all cultures. In marketing information, terms of politeness might be appropriate. Use the imperative mood in the first sentence of each step. 

2. When referring to a range of version numbers:
- Do not use `above / up` version ❌
Reason: version numbers should be essentially viewed as timeline markers rather than quantity markers.

- Use earlier / later version ✅
    `higher` version is not wrong and it is used in some product manuals, however, it is recommended to use `later` in Pulsar documentation to standardize on one expression for greater consistency in presenting the information to users.❗️

These rules are documented in https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.tdt6lehtxogm

cc @momo-jun @DaveDuggins @D-2-Ed

-->

> Note: this lua script may cause crash in the newest version of Wireshark, see [#18439](https://github.com/apache/pulsar/issues/18439). 
> Please make sure the version of Wireshark is lower than v4.0.0.

Reviewer 2: ```suggestion
**Note**: this Lua script may cause a crash in the newest version of Wireshark, see https://github.com/apache/pulsar/issues/18439.  Please make sure the version of Wireshark is lower than v4.0.0.
```

Make use of GitHub Markdown flavor.

Reviewer 2: Fine. Then I'll remove the links magic but retain the bold trick as there's no breaking regression:

```suggestion
> **Note**:this Lua script may cause a crash in the newest version of Wireshark, see [#18439](https://github.com/apache/pulsar/issues/18439). Please make sure the version of Wireshark is lower than v4.0.0.
```
